docs/azure_arc_jumpstart/azure_arc_k8s/day2/calico/_index.md

```diff
@@ -0,0 +1,127 @@
+
+![Showing pods deployment](./05.png)
+
+
+## Deploy a customized Azure policy to verify if calicocloud/calico enterprise components enabled in your Azure Arc Kubernetes cluster.
+
+* In the Search bar, look for *Policy* and click on *Definitions* which will show you all of the available Azure policies.
+
+    ![Searching for Azure Policy definitions](./06.png)
+
+    ![Searching for Azure Policy definitions](./07.png)
+
+* Click on *+ Policy definition* to add a new policy. 
+
+    ![Create a Azure policy for Kubernetes cluster](./08.png)
+
+    ![Create a Azure policy for Kubernetes cluster](./09.png)
+
+* In the above example, the *Definition location* of the policy represent the Azure subscription. Also, make sure *Category* is set to *Kubernetes*. We will use `audit` as default effect for this policy. 
+
+  You can either copy the `calicoapipolicy.json` file located in this folder into `POLICY RULE` or create a policy directly with azure cli with below command.
+
+  ```shell
+  az policy definition create --name 'arcdemo-calicoapi-policy' --display-name 'Audit Calico api server' --description 'This policy audit that if calicocloud/calico enterprise components exist' --rules 'calicoapipolicy.json' --mode All
+  ```
+
+* Once the policy is been created, you should be able to find it in *Policy | Definitions* 
+  
+  ![Calico Azure policy ](./10.png) need change
+    
+
+* Click the policy you created, and assign it to your resource group which include the arc clusters. Alternatively, you can assign the policy to entre Azure subscription.
+  
+  ![Assign Calico Azure policy ](./11.png) need change
+  
+
+* After the assignment, the policy task will start the evaluation against arc enabled cluster under your resource group. If you have 2 clusters installed calicocloud/calico enterprise, it will show as  "compliant" with `2 out of 2 ` (The gatekeeper will look for deployment with label 'apiserver' as audit labellist). To check this, go back to the main Policy page in the Azure portal.
+
+    > **Note: The process of evaluation against the cluster can take 30min.**
+
+    ![Azure policy evaluation](./12.png)
+
+    > If you create ks8 without enabling azure policy, then the audit result of this policy will be `0 out of 0` as `Compliant`
+
+
+## Clean up environment
+
+Complete the following steps to clean up your environment.
+
+* From the Policy page in the arc eks/gke portal, disable the extension of Azure policy under "Onboard to Azure Policy for Azure Arc enabled Kubernetes clusters​"
+    ![Disable Azure Policy addon](./13.png)
+    ![Disable Azure Policy addon](./14.png)
+
+
+* Delete the cluster as described in the [Terraform teardown instructions for eks](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/eks/eks_terraform/) or [Terraform teardown instructions for gke](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/gke/gke_terraform/) .
```

Comment:
Please refer to the full Jumpstart scenarios names as described in my comment at the beginning of the README.

Reply:
changed